src/routes.ts

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 import express = require('express');
 const router = express.Router();
-import { version } from '../package.json';
+// import { version } from '../package.json';
```

Comment:
@PanopticaRising 😞 
Please uncomment temporary fixes after they are merged in...